src/index.ts

```diff
@@ -92,6 +92,18 @@ export const FileSystem = {
     return FileAccessNative.fetch(resource, init);
   },
 
+  /**
+   * Return the local storage directory for app groups.
+   *
+   * This is an Apple only feature.
+   */
+  getAppGroupDir(groupName: string) {
+    if (Platform.OS !== 'ios' && Platform.OS !== 'macos') {
+      return Promise.reject('AppGroups are available on Apple devices only');
```

Comment:
Please wrap string in an `Error` object.

Reply:
Yep, will do.